pkgs/shells/nushell/plugins/polars.nix

```diff
@@ -1,20 +1,20 @@
-{
-  stdenv,
-  lib,
-  rustPlatform,
-  openssl,
-  nushell,
-  pkg-config,
-  IOKit,
-  Foundation,
-  nix-update-script,
+{ stdenv
+, lib
+, rustPlatform
+, openssl
+, nushell
+, pkg-config
+, IOKit
+, Foundation
+, nix-update-script
+,
```

Comment:
If you're going to reformat code, please do so in a separate commit from any other changes.